qt/scientific_interfaces/EnggDiffraction/EnggDiffGSASFittingViewQtWidget.cpp

```diff
@@ -10,6 +10,13 @@ namespace CustomInterfaces {
 EnggDiffGSASFittingViewQtWidget::EnggDiffGSASFittingViewQtWidget() {
   setupUI();
 
+  m_zoomTool.reset(
+      new QwtPlotZoomer(QwtPlot::xBottom, QwtPlot::yLeft,
+                        QwtPicker::DragSelection | QwtPicker::CornerToCorner,
+                        QwtPicker::AlwaysOff, m_ui.plotArea->canvas()));
```

Comment:
Don't use `reset` here. Use `make_unique` for [these reasons](https://stackoverflow.com/questions/37514509/advantages-of-using-make-unique-over-new-operator).